core/frontend/src/extension/ExtensionAdmin.ts

```diff
@@ -49,13 +47,9 @@ export class ExtensionAdmin {
    * @internal
    */
   public onStartup = async () => {
-    await this.activateExtensionEvents(ActivationEvent.onStartup);
+    await this.activateExtensionEvents("onStartup");
```

Comment:
This should use the type instead of the string directly.

Reply:
Can you elaborate? Afaik string union types can't be used as values, unless I make constants to go with it, but then I'd have to import a value from Extension.ts (which I was hoping to avoid) instead of only types

Reply:
fair point :)  I was still thinking from the perspective it was a enum or constant but with the union you're right.